mantidproject/mantid
# Reorganise Release Checklist

**Description of work.**
This PR reorganises the release checklist so that it is split into a number of checklists based on the role of each of the people involved in the release.

Code snippets for running scripts in the technical checklist will be added in a later PR.

**To test:**
Read through the checklist and make sure it is making sense and is clear. Make sure that each role has the correct points in their checklist.

Make sure the links all work by building the dev-docs target and opening the generated html file.

Fixes #30663

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
